crates/uv/src/commands/pip/install.rs

```diff
@@ -176,7 +176,7 @@ pub(crate) async fn pip_install(
     // Ideally, the resolver would be fast enough to let us remove this check. But right now, for large environments,
     // it's an order of magnitude faster to validate the environment than to resolve the requirements.
     if reinstall.is_none() && upgrade.is_none() && source_trees.is_empty() && overrides.is_empty() {
```

Comment:
Looks like `overrides` is empty here?

Reply:
Yeah I think `satisfies` doesn't quite work with overrides as-is.